src/main/kotlin/dev/restate/sdktesting/junit/TestSuite.kt

```diff
@@ -54,7 +54,10 @@ class TestSuite(
 
     // Apply additional runtime envs
     val restateDeployerConfig =
-        getGlobalConfig().copy(additionalRuntimeEnvs = additionalEnvs, restateNodes = restateNodes)
+        getGlobalConfig()
+            .copy(
+                additionalRuntimeEnvs = additionalEnvs + Pair("DO_NOT_TRACK", "true"),
```

Comment:
Can you set this instead in the init block of `RestateContainer` (look around line 126)? That's where I set all the "static" envs for the restate container